src/theme/index.tsx

```diff
@@ -280,8 +456,4 @@ html {
 a {
  color: ${({ theme }) => theme.deprecated_blue1}; 
 }
-
-:root {
-  ${({ theme }) => (theme.darkMode ? cssStringFromTheme(darkTheme) : cssStringFromTheme(lightTheme))}
-}
```

Comment:
This code is needed and shouldn't be removed as it's what adds NFT/vanilla-extract styles to the page